cmd/rook/edgefs/edgefs.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
+//Cmd is the main command for edgefs operator and daemons
```

Comment:
could you make a space after the start of the comment line? `// Cmd...`

Reply:
Sure.